### PR TITLE
Mario/2302 show tombstone validator

### LINKED ISSUE #2302: Show status if validator is tombstoned
**Is your feature request related to a problem? Please describe.**

as a user i'd like to see if a validator is tombstoned

**Describe the solution you'd like**

Get and display if the validator is tombstoned from the slashing_info, eg: 
```
index_offset: "34790"
jailed_until: "1970-01-01T00:00:00Z"
missed_blocks_counter: "0"
start_height: "0"
tombstoned: false <--
```

**Describe alternatives you've considered**

Add a red cross (+ or x ) on the status, as we do it with the rejected proposals

### PR DESCRIPTION
**Pull request by mariopino**
Closes #2302

**Description:**

Show INACTIVE status if validator status is either jailed or tombstoned. Show `detailed_status` as a subtitle in validator page.

Now both `status` and  `detailed_status`  are computed fields, when merge  https://github.com/luniehq/lunie-backend/pull/16, we'll simply use `status` and `detailed_status` directly as provided by the backend.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by mariopino** [comment]
Some screenshots:

Validator list:

image

Jailed:

image

Tombstoned:

image

File: src/components/staking/LiValidator.vue [hunk 1]
```diff
@@ -77,6 +89,23 @@ export default {
...
+      if (
+        this.validator.jailed ||
+        this.validator.tombstoned ||
+        this.validator.status === 0
+      )
+        return `Inactive`
+      return `Active`
+    },
+    status_detailed() {
+      if (this.validator.jailed) return `Temporally banned from the network`
```
Comment (jbibla) [inline src/components/staking/LiValidator.vue]: @faboweb we need to find a solution for spelling errors. perhaps we can make a PR for this Github app to check HTML.

https://github.com/apps/prosebot

File: src/components/staking/LiValidator.vue [hunk 2]
```diff
@@ -77,6 +89,23 @@ export default {
...
+      if (this.validator.tombstoned) return `Banned from the network`
+      if (this.validator.status === 0) return `Banned from the network`
```
Comment (jbibla) [inline src/components/staking/LiValidator.vue]: i don't think this is true - we'll have to improve some of these messages.

File: src/components/staking/LiValidator.vue [hunk 3]
```diff
@@ -77,6 +89,23 @@ export default {
...
+    status() {
+      if (
+        this.validator.jailed ||
+        this.validator.tombstoned ||
+        this.validator.status === 0
+      )
+        return `Inactive`
+      return `Active`
+    },
+    status_detailed() {
```
Comment (jbibla) [inline src/components/staking/LiValidator.vue]: this is duplicated. is it going to come from Hasura?

File: src/components/staking/LiValidator.vue (hunk 2 above)
Comment (mariopino) [reply src/components/staking/LiValidator.vue]: Why do you think it's not true?

File: src/components/staking/LiValidator.vue (hunk 3 above)
Comment (mariopino) [reply src/components/staking/LiValidator.vue]: Yes, both status and status_detailed.

File: src/components/staking/LiValidator.vue (hunk 2 above)
Comment (jbibla) [reply src/components/staking/LiValidator.vue]: i just mean they are not necessarily "banned"

Comment (mariopino) [reply src/components/staking/LiValidator.vue]: We can customize the messages in the new backend.

File: src/components/staking/LiValidator.vue (hunk 1 above)
Comment (faboweb) [reply src/components/staking/LiValidator.vue]: Please take care of that

**Pull request merged**